pkg/security/resolvers/resolvers.go

```diff
@@ -39,6 +39,11 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+// ResolversOpts defines common options
+type ResolversOpts struct {
+	PathResolutionEnabled bool
+}
```

Comment:
do we have to create this struct? it seems a bit awkward that `PathResolutionEnabled` opt is part of "common options for resolvers", since it is a very specific one for processResolver

Reply:
I do prefer to path a struct that will be use to tweak other resolvers later. I just used it for the process resolver but it should be used for the file path resolution as well